packages/dx-react-chart-demos/src/demo-sources/layout/bootstrap4/title.jsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import {
+  Chart,
+  ArgumentAxis,
+  ValueAxis,
+  BarSeries,
+  Legend,
+  Title,
+} from '@devexpress/dx-react-chart-bootstrap4';
+
+import { Stack } from '@devexpress/dx-react-chart';
+
+import { ageStructure } from '../../../demo-data/data-vizualization';
+
+const TextComponent = ({ text, ...restProps }) => (
+  <h3 {...restProps} className="w-100 text-left mb-3">
+    {text}
+  </h3>
+);
+
+export default class Demo extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      data: ageStructure,
+    };
+  }
+
+  render() {
+    const { data: chartData } = this.state;
+
+    return (
+      <div className="card">
+        <Chart
+          data={chartData}
+        >
+          <ArgumentAxis />
+          <ValueAxis />
+
+          <BarSeries
+            name="👶 Young"
+            valueField="young"
+            argumentField="state"
+          />
+          <BarSeries
+            name="🧑 Adult"
+            valueField="middle"
+            argumentField="state"
+          />
+          <BarSeries
+            name="🧓 Old"
+            valueField="older"
+            argumentField="state"
+          />
+          <Stack
+            stacks={[
```

Comment:
Extract it to a constant (false plugin render).

Reply:
ok